definitions/family-private/json/CaseField.json

```diff
@@ -770,5 +770,250 @@
     "Label": "Other safety or welfare concerns details",
     "FieldType": "TextArea",
     "SecurityClassification": "Public"
+  },
+
+
+
+  {
+    "LiveFrom": "23/08/2021",
+    "CaseTypeID": "C100",
+    "ID": "ApplicantAttendedMIAMLabel",
+    "Label": "## Has the applicant attended a Mediation information & Assessment Meeting (MIAM) in the past 4 months?",
```

Comment:
That's the label as it is in FR. Is that exactly how we want it to be for C100 as well (i.e. same text)?

Reply:
The only AC I've got is "do it exactly the same way it's in FR".

Reply:
Oh okay aha :D